vnext/Libraries/Text/Text.windows.js

```diff
@@ -0,0 +1,374 @@
+/**
```

Comment:
I'm guessing that these files are not supposed to be checked in since they're under the generated Library directory instead of under src but I'm not sure if I'm supposed to be adding them to .gitignore?

Reply:
Figured it out